input-explainer.md

```diff
@@ -336,29 +336,29 @@ The UA may update the `gamepad` state at any point, but it must remain constant
 
 ### XR gamepad mapping
 
-The WebXR Device API also introduces a new standard controller layout indicated by the `mapping` value of `xr-standard`. (Additional mapping variants may be added in the future if necessary.) This defines a specific layout for the inputs most commonly found on XR controller devices today. The following table describes the buttons/axes and their physical locations:
+The WebXR Device API also introduces a new standard controller layout indicated by the `mapping` value of `xr-standard`. (Additional mapping variants may be added in the future if necessary.) This defines a specific layout for the inputs most commonly found on XR controller devices today. The following table describes the buttons/axes and their associated physical inputs:
 
-| Button     | `xr-standard` Location            |
-| ---------- | --------------------------------- |
-| buttons[0] | Primary trigger                   |
-| buttons[1] | Primary Touchpad/Joystick click   |
-| buttons[2] | Grip/Secondary trigger            |
-| buttons[3] | Secondary Touchpad/Joystick click |
+| Button     | `xr-standard` Binding  |
+| ---------- | -----------------------|
+| buttons[0] | Primary trigger        |
```

Comment:
Above, `buttons[0]` is defined to possibly be a "primary button" instead.  We should at least mention "Primary trigger/button" here.

Similar to the touchpad/thumbstick ambiguity, there is ambiguity across profiles that would be possible here if two controllers both had a button and a trigger, but differed on whether they considered the trigger or the button to be "primary".  However, since buttons and triggers are far more interchangeable than touchpads and thumbsticks, I'm less worried there.  My gut is that, as we work out the actual generic profiles you sketched out for today's controllers, this ambiguity won't prove to be a problem.